packages/api/src/bridge/token-data.ts

```diff
@@ -0,0 +1,16 @@
+import { JwtPayload } from "../authentication/types/jwt-payload.js";
+
+let getOutsideJwtPayload: (() => JwtPayload | null) | null = null;
+
+export function setJwtPayloadFunction(fn: () => JwtPayload | null): void {
```

Comment:
This method could create many issue if there is race condition.

Reply:
I think, this method shouldn't be used in concurrent code